### ISSUE TITLE
CIGAR string "M" cannot be recognized

### ISSUE THREAD

**Issue by wolfylair** [schneebergerlab/syri#27]
Hi,

I was running SyRI to look for variations and rearrangement, but SyRI didn't recognize my BAM file. I have deleted all small contigs and only kept the ones in chromosome length from both reference sequence and query sequence, based on the error #15.

Here is my command and the error:

```
syri -c test.bam -r ../../../2.Alignments/1.ref/hs.chr.fa -q ../../mr-2k.big.fasta -k -F B
syri - WARNING - starting
Reading Coords - ERROR - Error in reading the BAM file
```
Could you please help me figure out the issue? I can provide more details if you need more.

Thank you so much for your time!

Regards

**Comment by mnshgl0110** [comment]
Hi, Could you please run SyRI with `--log DEBUG` and share the syri.log file.

**Comment by wolfylair** [comment]
Hi,
The attachment is the log file. 

[syri.log](https://github.com/schneebergerlab/syri/files/4468994/syri.log)

Regards

**Comment by mnshgl0110** [comment]
In the log file, I do not see anything specific. However, it seems that an older version of SyRI is being used. Could you please use the latest version as that have better debugging information.

**Comment by wolfylair** [comment]
Hi,

I reinstalled the syri v1.1 from the released tar ball. And here is the log file for v1.1
[syri.log](https://github.com/schneebergerlab/syri/files/4469072/syri.log)

**Comment by mnshgl0110** [comment]
v1.1 is already quite old now and do not have all of the debugging information. I suggest you to install from the current master branch.

**Comment by wolfylair** [comment]
Hi,

I got the one from master branch. And here is the log file.
[syri.log](https://github.com/schneebergerlab/syri/files/4469271/syri.log)
Now it turned out to be a CIGAR issue. It seemed that syri couldn't recognize the string 'M'

**Comment by mnshgl0110** [comment]
Yes. This is input file requirement. If you are using minimap2 for alignments, then you can use `--eqx` to get CIGAR string in correct format.
https://schneebergerlab.github.io/syri/pipeline.html

**Comment by wolfylair** [comment]
Hi, i just noticed that this issue have been solved in #20 . I think i am good now, thank you so much for your time and patience!
